File: HPWorkpath_v1.6.3/Samples/ExampleAPIServices/source/DeviceUsageSample/src/com/hp/workpath/sample/deviceusagesample/fragment/ResponseInterface.java
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.deviceusagesample.fragment;

import com.hp.workpath.api.Result;
import com.hp.workpath.api.deviceusage.DeviceUsageInfo;

public interface ResponseInterface {
    void success(DeviceUsageInfo info);
    void failure(String msg, Result result);
}

File: HPWorkpath_v1.6.3/Samples_Kotlin/ExampleAPIServices/source/DeviceUsageSample/src/com/hp/workpath/sample/deviceusagesample/fragment/ResponseInterface.kt
// Copyright 2018 HP Development Company, L.P.
// SPDX-License-Identifier: MIT
package com.hp.workpath.sample.deviceusagesample.fragment

import com.hp.workpath.api.Result
import com.hp.workpath.api.deviceusage.DeviceUsageInfo

interface ResponseInterface {
    fun success(info: DeviceUsageInfo?)
    fun failure(msg: String?, result: Result?)
}